geoapps/io/constants.py

```diff
@@ -0,0 +1,256 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoapps.
+#
+#  geoapps is distributed under the terms and conditions of the MIT License
+#  (see LICENSE file at the root of this source code package).
+
+
+required_parameters = ["inversion_type"]
```

Comment:
I would think that `data`, `topography` and `mesh` would be also base requirements